tox.ini

```diff
@@ -5,11 +5,11 @@
 
 [tox]
 envlist =
-	py27-django1.{4,8}
+	py27-django1.{5,8}
 
 [testenv]
 commands = {envpython} run_tests.py
 deps =
 	django_nose
-	django1.4: Django>=1.4, <1.5
+	django1.5: Django>=1.5, <1.6
```

Comment:
I don't see any reason to support django 1.5 either. Might as well drop everything but 1.8